android/build.gradle

```diff
@@ -56,7 +73,5 @@ repositories {
 }
 
 dependencies {
-  //noinspection GradleDynamicVersion
```

Comment:
this is a regression, from a lint perspective. The IDE will now flag this line as a dynamic version I think, whereas we want to ignore it?